Bed.java

```diff
@@ -2,11 +2,12 @@ public class Bed {
     private double hight;
     private int PillowAmount;
```

Comment:
```suggestion
    private int pillowAmount;
```
the convention in java is first letter small and the first letter on each word after the first big letter